src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -52,10 +52,10 @@ namespace big
 			{
 				if (bplyr->block_join)
 				{
-					((player_command*)(command::get(RAGE_JOAAT("breakup"))))->call(bplyr, {});
+					dynamic_cast<player_command*>(command::get(RAGE_JOAAT("breakup")))->call(bplyr, {});
 					g_notification_service->push("Player Kicked due to Block Join bypass",
 					    std::format("{} was kicked", net_player_data->m_name));
-					LOG(WARNING) << net_player_data->m_name << "(" << net_player_data->m_gamer_handle.m_rockstar_id << ") has been kicked(breakup) due to block join bypass"
+					LOG(WARNING) << net_player_data->m_name << "(" << net_player_data->m_gamer_handle.m_rockstar_id << ") has been kicked(breakup) due to block join bypass";
```

Comment:
Could you maybe rewrite this text? Nobody will understand why a bypass is needed. Maybe say like "Block join method failed, sending breakup kick instead."

Reply:
Alright, I will do this